src/oom/oomd-manager.c

```diff
@@ -125,6 +125,10 @@ static int process_managed_oom_message(Manager *m, uid_t uid, JsonVariant *param
                 ctx = hashmap_get(monitor_hm, empty_to_root(message.path));
                 if (ctx)
                         ctx->mem_pressure_limit = limit;
+
+                /* Disable wake-ups for "ManagedOOMSwap" unless entries are present. */
+                sd_event_source_set_enabled(m->swap_context_event_source,
```

Comment:
it's ok to ignore the return value here I guess. but please make that explicit for the sake of static analyzers, i.e. cast the result of the call to `(void)`.

(alternatively, write a debug log message in case of failure and proceed, too)

Reply:
I'll just log, addressed in most recent patch.